.github/workflows/doc.yml

```
@@ -71,6 +71,8 @@ jobs:
             sudo apt-get -y update
             sudo apt-get -y install \
                 cmake graphviz ninja-build doxygen nlohmann-json3-dev
+            sudo npm install -g @mermaid-js/mermaid-cli
+            sudo npm -g rebuild puppeteer
```

```suggestion
            sudo npm -g rebuild puppeteer
            sudo npx puppeteer browsers install chrome-headless-shell
```
or something like that ... the doc fails now with:
```
Error: Could not find Chrome (ver. 131.0.6778.204). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npx puppeteer browsers install chrome-headless-shell`) or
 2. your cache path is incorrectly configured (which is: /home/runner/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.
```

---

Yeah... trying to figure this one out, didn't realize it was going to be so complicated. Apparently `mmdc` requires enough styling to require a local copy of chromium to work...